plugin/plugins/voiceprint/__init__.py

```
@@ -0,0 +1,500 @@
+            return Err({"error": f"声纹比对失败: {str(e)}"})
+
+    @plugin_entry(
+        id="set_threshold",
+        name="设置阈值",
+        description="设置声纹识别的相似度阈值",
+        input_schema={
+            "type": "object",
+            "properties": {
+                "threshold": {
+                    "type": "number",
+                    "description": "阈值 (0.0-1.0)，越低越严格",
```

Ruff (0.15.7)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)